controllers/ManagementController.php

```diff
@@ -101,6 +131,18 @@ private function checkIfUserExist($id)
         throw new $unauthorizedException();
     }
 
+    /**
+     * @param int $id
+     * @return SocialMediaPermission|null
+     */
+    private function chekcIfUserPermission($id)
```

Comment:
This function name is incorrect: I think we should name it "checkIfPermissionExists" to be consistent with checkIfUserExist function

Reply:
you're right